app/src/main/java/com/eventyay/organizer/core/main/MainActivity.java

```diff
@@ -39,15 +43,26 @@
 
     public static final String EVENT_KEY = "event";
     private long eventId = -1;
+
     private final List<Integer> drawerItems = Arrays.asList(
-        /*R.id.nav_feedback,
+        R.id.nav_dashboard,
+        R.id.nav_attendees,
+        R.id.nav_share,
+        R.id.nav_about_event,
+        R.id.nav_event_settings);
+
+    private final List<Integer> drawerExtraItems = Arrays.asList(
+        R.id.nav_sell,
+        R.id.nav_orders,
+        R.id.nav_tickets,
+        R.id.nav_edit_event,
```

Comment:
Why edit event is not in the main drawer list? Is it what we want?

Reply:
Yes it has to be hidden for now.